.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Release pipeline
+
+on:
+  release:
+    types:
+      - released
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  INTEGRATION: "haproxy"
+  ORIGINAL_REPO_NAME: 'newrelic/nri-haproxy'
+  TAG: ${{ github.event.release.tag_name }}
+
+jobs:
+
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@main
+        env:
+          WORKDIR: /home/gha
+          AWS_S3_BUCKET_NAME: "nr-downloads-main"
+          AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock"
+          AWS_REGION: "us-east-1"
```

Comment:
why are these passed as env ?

Reply:
just to not have them directly in the configuration of the action to be consistent with other cases. We should decide if we want to use env vars for everything or put values directly in the action config

Reply:
I've changed to use env vars for everything

Reply:
I'm not amused with the mix, but at this point I just want to let this see the light so I'm dropping my perfectionism for once :(

Reply:
now it's consistent! :) it's all in env vars :)